Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -282,6 +283,7 @@ namespace AtomSampleViewer
         SampleComponentManager::RegisterSampleComponent(SampleEntry::NewRPISample( "RPI/SceneReloadSoakTest", azrtti_typeid<SceneReloadSoakTestComponent>() ));
         SampleComponentManager::RegisterSampleComponent(SampleEntry::NewRPISample( "RPI/Shading", azrtti_typeid<ShadingExampleComponent>() ));
         SampleComponentManager::RegisterSampleComponent(SampleEntry::NewRPISample( "RPI/StreamingImage", azrtti_typeid<StreamingImageExampleComponent>() ));
+        SampleComponentManager::RegisterSampleComponent(SampleEntry::NewRPISample( "RPI/ShaderReloadSoakTest", azrtti_typeid<ShaderReloadSoakTestComponent>() ));
```

Comment:
I don't think the sample should be called "ShaderReloadSoakTest" because the sample itself doesn't soak anything. It just provides buttons to apply shader changes. I would just call it "ShaderReloadTest" or "FullscreenShaderReloadTest". You can still call the _script_ "ShaderReloadSoakTest.bv.lua" because that does do soaking.

Reply:
Good point.